styles.css

```diff
@@ -0,0 +1,391 @@
+*{
+	box-sizing: border-box;
+}
+
+body {
+    max-width: 80rem;
+    margin: auto;
+}
+
+
+header {
+	height:10%;
+	background-color:#093C60;
+	color:white;
+    padding: 5% 13%;
+    max-height: 400px;
+    position: relative;
+}
+
+header h1 {
+	font-size: 1.1em;
+	font-family: 'Courgette', cursive;
+	margin-top: -10px;
+    margin-left: -65px;
+    padding-left: 30px;
+    color: #75B441;
+}
+
+.header-logo {
+    max-width: 40%;
+    min-width: 20rem;
+    height: auto;
+    margin-left: -75px;
+    padding-left: 30px;
+}
+
+.cart-img {
+    display: inline;
+    max-width: 30px;
+    height: auto;
+    position: absolute;
+    top: 40px;
+    right: 100px;
+}
+
+.checkout-button {
+    background-color: #81B741;
+    border: none;
+    color: white;
+    padding: 10px 20px;
+    text-align: center;
+    font-family: 'Work Sans', sans-serif;
+    font-weight: 700;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 13px;
+    width: 12%;
+    border-radius: 2px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    display: block !important;
+    position: absolute;
+    top: 70px;
+    right: 100px;
+}
+
+nav {
+	background-color: #ECECEC;
+    max-height: 5rem;
+    padding-left: 30px;
+    padding: 2em 13%;
+	margin-bottom: 3%;
+}
+
+nav a {
+	text-decoration: none;
+	color: #093C60;
+	font-family: 'Work Sans', sans-serif;
+	font-size: 1.2rem;
+    letter-spacing: 1px;
+    font-weight: 700;
+}
+
+nav a:visited {
+	text-decoration: none;
+}
+
+.main-content {
+    overflow: hidden;
+    margin-bottom: 25px;
+}
+
+.main-content img {
+    float: left;
+    padding: 0 35px 20px 0;
+}
+
+section {
```

Comment:
The combo of margin right and padding left is a little wonky. Maybe set a max width on the section and margin: 0 auto, which should center it.